packages/cozy-stack-client/src/NextcloudFilesCollection.js

```diff
@@ -7,11 +7,13 @@ const NEXTCLOUD_FILES_DOCTYPE = 'io.cozy.remote.nextcloud.files'
 const normalizeDoc = DocumentCollection.normalizeDoctypeJsonApi(
   NEXTCLOUD_FILES_DOCTYPE
 )
-const normalizeNextcloudFile = (sourceAccount, parentPath) => file => {
+const normalizeNextcloudFile = (sourceAccount, parentPath, options) => file => {
```

Comment:
It's a subjective, but I tend to prefer being explicit in the function signatures, like `(sourceAccount, parentPath, {fromTrash})` rather than the generic `options`. This way, I know exactly what parameters are expected.